bayes_opt/helpers.py

```diff
@@ -10,11 +10,13 @@ class UtilityFunction(object):
     An object to compute the acquisition functions.
     """
 
-    def __init__(self, kind='ucb', kappa=1.96):
+    def __init__(self, kind, kappa, xi):
```

Comment:
I think it makes sense to give both kappa and xi default values so when the object is instantiated with a particular acquisition function in mind, the user doesn't have to bother with an unrelated parameter.